src/config/AppState.js

```diff
@@ -0,0 +1,86 @@
+import {
+  COMMON_DOTS,
+  COMMON_DOTS_1,
+  COMMON_DOTS_2,
+  COMMON_DOTS_3,
+  COMMON_DOTS_4,
+  COMMON_DOTS_5,
+  COMMON_DOTS_6,
+  COMMON_DOTS_7,
+  COMMON_DOTS_8,
+  COMMON_DOTS_9,
+  COMMON_DOTS_10,
+  MAX_LH,
+} from './constants';
+
+export const AppState = {
+  postOvulation: false,
+  preOvulation: true,
+  ovulation: false,
+  ovulationActive: false,
+  postOvulationActive: false,
+  preOvulationActive: false,
+  // we delay five seconds during the 13 and 14 days
+  delay: 5,
+  dayCount: 1, // we initialize the day counter to 1
+  secretLhFsh: false, // at the begening we do not secret any hormones
+  secretProgest: false,
+  secretOestro: false,
+  // starting state for the progesterone dots
```

Comment:
@azizmashkour, for this comment I meant to explain what the set of dots means, not that this was the start of the code (no need for that or the "end" comment). Just explain what this array of points is, (position on canvas at the beginning or something else).